internal/packages/packages.go

```diff
@@ -223,5 +237,5 @@ func isDataStreamManifest(path string) (bool, error) {
 	if err != nil {
 		return false, errors.Wrapf(err, "reading package manifest failed (path: %s)", path)
 	}
-	return m.Title != "" && (m.Type == "logs" || m.Type == "metrics"), nil
+	return m.Title != "" && (m.Type == dataStreamTypeLogs || m.Type == dataStreamTypeMetrics), nil
```

Comment:
nit: Actually this function isn't perfect. If there is another dataset type, it will simply fail (OCP broken). I wonder if there is a different way to recognize the DataStream. If you find any, feel free to adjust the implementation.

Reply:
Yeah, it's not perfect but I think this check here might have some good side-effects. In the asset test runner we check if we are dealing with a `logs` data stream and, if so, check that ingest pipeline assets have also been loaded. Similarly, whenever a new data stream type is added in the future we might need to have special handling for it in `elastic-package`. So being strict here about which data stream types `elastic-package` knows how to handle may be a good thing. 

BTW, I was checking the data stream manifest spec to see if we have defined the data stream `type` field as an `enum` and we have. However, while doing this I noticed that there is a data stream type we have not considered in `elastic-package`: [`traces`](https://github.com/elastic/package-spec/blob/4e314c0a650da34a8f2d6f104dd659664b5377f8/versions/1/data_stream/manifest.spec.yml#L117-L120). It was added for APM in https://github.com/elastic/package-spec/pull/78. I'm guessing the team is not using `elastic-package` yet so they are not running into any issues with this type not being supported. However, I've created https://github.com/elastic/elastic-package/issues/223 to track adding support for this type in `elastic-package`.